### ISSUE TITLE
[Feature request] Network disconnected means that - disconnected

### ISSUE THREAD

**Issue by elad335** [RPCS3/rpcs3#14057]
### Quick summary

When using network disconnected mode, the PS3 application should not be able to listen on any real port if the PC and loopback oavkets should not be visible to host operating system programs. It should be a closed virtual space which does not interact even with host computer.

**Comment by digant73** [comment]
IMO this FR is absolutely logical and something that I considered already implemented.

**Comment by elad335** [comment]
It's not implemented in sys_bnet_net syscalls I am pretty sure. Some redirection is needed for public IPs I believe is what needs to be done here. Same with DNS handling.

**Comment by digant73** [comment]
yeah, I think this FR should be considered to be implemented soon
